scripts/proxy-for-jupyter/on-start.sh

```diff
@@ -0,0 +1,34 @@
+# This script configures proxy settings for your Jupyter notebooks and the SageMaker notebook instance.
+# This is useful for use cases where you would like to configure your notebook instance in your custom VPC
+# without direct internet access to route all traffic via a proxy server in your VPC.
+
+# Please ensure that you have already configure a proxy server in your VPC.
+
+#!/bin/bash
+ 
+set -e
+
+su - ec2-user -c "mkdir /home/ec2-user/.ipython/ && mkdir /home/ec2-user/.ipython/profile_default/ && mkdir /home/ec2-user/.ipython/profile_default/startup/ && touch /home/ec2-user/.ipython/profile_default/startup/00-startup.py"
```

Comment:
This command can be reduced to 
```
su - ec2-user -c "mkdir -p /home/ec2-user/.ipython/profile_default/startup/ && touch /home/ec2-user/.ipython/profile_default/startup/00-startup.py"
```

`-p` option in mkdir create intermediate directories as required.

Reply:
Thanks for your suggestion. I have made the necessary changes to the script